### PR TITLE
Adds an option (--alfred-file) for existing alfred json files

### PR DESCRIPTION
**Pull request by mruettgers**
To be able to create a nodes.json without an alfred instance running in master mode I added an option (--alfred-file) to bat2nodes.py that allows the processing of an existing alfred json file (e.g. ffmap.freifunk-rheinland.net/alfred.json).

### CODE REVIEW

File: bat2nodes.py [hunk 1]
```diff
@@ -46,7 +51,7 @@
 
 for mesh_interface in options['mesh']:
   bm = batman(mesh_interface)
-  db.parse_vis_data(bm.vis_data(options['alfred']))
+  db.parse_vis_data(bm.vis_data(options['alfred'] or options['alfred_file']))
```
Comment (tcatm) [inline bat2nodes.py]: I'm not sure this will work. Have you tested it?

Comment (mruettgers) [inline bat2nodes.py]: Yes, of course :-).

**Comment by mruettgers** [comment]
State.json is stored in the current working directory.
I think a better approach is to store the file in the given destination directory.

**Comment by kokel** [comment]
The state.json is needed only in ffmap-backend to load the last state in the next run of bat2nodes.py. Why store state.json in ffmap-d3 while it won't use it?

**Comment by mruettgers** [comment]
Hm, ok, that's a point.
I was just wondering about those state.json files lingering around everywhere.
So I have to modify my script to chdir before calling bat2nodes.py.

**Pull request closed without merging**